plutus-core/plutus-core/src/PlutusCore/Default/Universe.hs

```diff
@@ -359,10 +358,6 @@ So, what to do? We adopt the following strategy:
 
 Doing this effectively bans builds on 32-bit systems, but that's fine, since we don't care about
 supporting 32-bit systems anyway, and this way any attempts to build on them will fail fast.
-
-Note: we couldn't fail the bounds check with 'AsUnliftingError', because an out-of-bounds error
-is not an internal one -- it's a normal evaluation failure, but unlifting errors
-have this connotation of being "internal".
```

Comment:
That distinction between internal and external errors is long gone, although I do need to remove its remnants from the code.